src/v/cluster/metadata_dissemination_service.cc

```diff
@@ -270,19 +270,13 @@ void metadata_dissemination_service::collect_pending_updates() {
             // Topic metadata is not there anymore, partition was removed
             continue;
         }
-        auto non_overlapping = calculate_non_overlapping_nodes(
```

Comment:
I think this was the only usage of the function, so it could be removed to avoid leaving dead code behind

Reply:
good point.

Reply:
were you planning to remove the dead code?

Reply:
done